code/game/objects/items/weapons/cigs.dm

```diff
@@ -221,26 +221,96 @@ LIGHTERS ARE IN LIGHTERS.DM
 
 
 /obj/item/clothing/mask/cigarette/menthol
+	desc = "A roll of tobacco and nicotine in a black and blue paper. Smells... minty."
+	item_state = "cigoff_menthol"
+	icon_state = "cigoff_menthol"
+	icon_on = "cigon_menthol"
+	icon_off = "cigoff_menthol"
+	type_butt = /obj/item/cigbutt/menthol
 	list_reagents = list("nicotine" = 40, "menthol" = 20)
 
 /obj/item/clothing/mask/cigarette/random
 
 /obj/item/clothing/mask/cigarette/random/New()
+	item_state = "cigoff_shadyjims"
+	icon_state = "cigoff_shadyjims"	
+	icon_on = "cigon_shadyjims"
+	icon_off = "cigoff_shadyjims"
 	list_reagents = list("nicotine" = 40, pick("fuel","saltpetre","synaptizine","green_vomit","potass_iodide","msg","lexorin","mannitol","spaceacillin","cryoxadone","holywater","tea","egg","haloperidol","mutagen","omnizine","carpet","aranesp","cryostylane","chocolate","bilk","cheese","rum","blood","charcoal","coffee","ectoplasm","space_drugs","milk","mutadone","antihol","teporone","insulin","salbutamol","toxin") = 20)
 	..()
 
 /obj/item/clothing/mask/cigarette/syndicate
+	desc = "A roll of tobacco and nicotine in a cool but suspicious black and red paper. Smells... weird, but it makes your nose feel better."
+	item_state = "cigoff_syndicate"
+	icon_state = "cigoff_syndicate"
+	icon_on = "cigon_syndicate"
+	icon_off = "cigoff_syndicate"
+	type_butt = /obj/item/cigbutt/syndicate
 	list_reagents = list("nicotine" = 40, "omnizine" = 20)
 
 /obj/item/clothing/mask/cigarette/medical_marijuana
+	desc = "A roll of marijuana in a medical white paper with a blue band. Smells... calming."
+	item_state = "cigoff_med"
+	icon_state = "cigoff_med"
+	icon_on = "cigon_med"
+	icon_off = "cigoff_med"
+	type_butt = /obj/item/cigbutt/med
 	list_reagents = list("thc" = 40, "cbd" = 20)
 
 /obj/item/clothing/mask/cigarette/robustgold
+	desc = "A premium roll of of tobacco and nicotine in a fancy gold paper. Smells a bit like wealth... and cancer."
+	item_state = "cigoff_gold"
+	icon_state = "cigoff_gold"
+	icon_on = "cigon_gold"
+	icon_off = "cigoff_gold"
+	type_butt = /obj/item/cigbutt/gold
 	list_reagents = list("nicotine" = 40, "gold" = 1)
+	
+/obj/item/clothing/mask/cigarette/robust
+	desc = "A roll of of tobacco and nicotine in a white paper with a red filter. Smells... robust. Nothing special about this one otherwise."
```

Comment:
```suggestion
	desc = "A roll of tobacco and nicotine in a white paper with a red filter. Smells... robust. Nothing special about this one otherwise."
```